### PR TITLE
DOCSP-35841: Monitoring pages for command + pool

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-35841>
Staging:
- [cmd monitoring](https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-35841-monitoring-pages/fundamentals/monitoring/command-monitoring/)
- [connection pool monitoring](https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-35841-monitoring-pages/fundamentals/monitoring/connection-monitoring/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/monitoring/command-monitoring.txt [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+       "ServerConnectionID": ...,
+       "ServerConnectionID64": ...,
+       "ServiceID": null,
+       "Reply": "..."
+   }
+
+CommandFailedEvent
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
```
Comment (rachel-mack) [inline source/fundamentals/monitoring/command-monitoring.txt:116]: For consistency with the other examples:

```suggestion
.. code-block:: bash
```

File: source/fundamentals/monitoring/connection-monitoring.txt [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+     - Description
+
+   * - ``ConnectionPoolCreated``
+     - Created when a connection pool is created.
+
+   * - ``ConnectionPoolReady``
+     - Created when a connection pool is ready.
+
+   * - ``ConnectionPoolCleared``
+     - Created when a connection pool is cleared.
```
Comment (rachel-mack) [inline source/fundamentals/monitoring/connection-monitoring.txt:73]: I'm guessing this means all operations have been completed/the pools is empty - could you clarify this?

File: source/fundamentals/monitoring/connection-monitoring.txt [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+Example Event Documents
+-----------------------
+
+The following sections show sample output for each type of connection
+pool monitoring event.
+
+ConnectionPoolCreated
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
```
Comment (rachel-mack) [inline source/fundamentals/monitoring/connection-monitoring.txt:115]: On the Command Monitoring and Cluster Monitoring pages you use ``bash``, while on this page only ``none`` - I'm not concerned about which you use, but you should be consistent.

```suggestion
.. code-block:: bash
```

File: source/fundamentals/monitoring/cluster-monitoring.txt [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+
+This guide shows you how to use the {+driver-short+} to monitor topology
+events in a MongoDB instance, replica set, or sharded cluster. The
+driver creates topology events, also known as Server Discovery and
+Monitoring (SDAM) events, when there are any changes in the state of the
+instance or cluster that you are connected to. 
+
+You can use information about topology changes in your
+application, or you can monitor cluster changes to learn more about
+how they affect your application.
```
Comment (rachel-mack) [inline source/fundamentals/monitoring/cluster-monitoring.txt:31]: You use this phrase on all three pages. How would you apply this information in your application? Could you make decisions about when to submit async requests? Could it influence batch sizes? Are there trends to look out for?

This is probably a tech review question, but without more information, this question doesn't really add anything.

**Review by rachel-mack — CHANGES_REQUESTED** [review]
A few comments:

File: source/fundamentals/monitoring/connection-monitoring.txt (hunk 3 above)
Comment (rustagir) [reply source/fundamentals/monitoring/connection-monitoring.txt:115]: fixed

File: source/fundamentals/monitoring/cluster-monitoring.txt (hunk 4 above)
Comment (rustagir) [reply source/fundamentals/monitoring/cluster-monitoring.txt:31]: My guess is that there are a lot of reasons why someone could enable monitoring. I'll do some research and add some detail to each of the pages.

**Review by rachel-mack — APPROVED** [review]
👍

**Pull request merged**